cmd/zstreamdump/zstreamdump.c

```diff
@@ -266,6 +309,13 @@ main(int argc, char *argv[])
 	}
 
 	fletcher_4_init();
+	int fp = -1;
+	fp = open("/tmp/adump", O_RDWR|O_CREAT|O_APPEND, O_RDWR);
```

Comment:
why we need this?

Reply:
oops... I missed this changes. It was for testing purpose. I will remove this.